apps/ui/src/Components/CommunityList/EmptyCommunityRides.tsx

```diff
@@ -0,0 +1,24 @@
+import { Card, CardContent, Typography } from '@mui/material';
+
+interface EmptyCommunityRidesProps {
+  communityTitle?: string;
+}
+
+const EmptyCommunityRides: React.FC<EmptyCommunityRidesProps> = ({
+  communityTitle,
+}) => {
+  const text = communityTitle
+    ? `Sorry, No rides available for community: "${communityTitle}" for now`
+    : 'Sorry, No rides available in the community for now';
```

Comment:
The second text option appears when there are no communities for the user, so I think a clearer text will be something like `Oops! It looks like you haven't joined any communities yet.`